src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ProvidePropertyAttribute.cs

```diff
@@ -72,6 +69,15 @@ public override object TypeId
                 return GetType().FullName + _propertyName;
             }
         }
+#else
```

Comment:
I don't think we should include the TypeId property right now - it won't be used, because it's not on a base class multiple attributes dervie from. For now, just keep it `#if FEATURE_ATTRIBUTE_TYPEID/#endif`